test/integration/lib/first-time.js

```diff
@@ -64,30 +64,6 @@ QUnit.test('render init screen', function (assert) {
     sandwich.click()
 
     return wait()
-  }).then(function() {
```

Comment:
I wish there was a way to preserve this test. Maybe the `ui-dev.js` file could have an integration-test version of locking?